firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigSharedPrefsClient.java

```diff
@@ -75,65 +83,75 @@ public class ConfigMetadataClient {
   private static final String REALTIME_BACKOFF_END_TIME_IN_MILLIS_KEY =
       "realtime_backoff_end_time_in_millis";
 
-  private final SharedPreferences frcMetadata;
+  /** Constants for custom signal limits.*/
+  private static final int CUSTOM_SIGNALS_MAX_KEY_LENGTH = 250;
```

Comment:
It will be part of our public documentation, for internal discussion it's documented here: [Link](https://docs.google.com/document/d/1BvpUiGqWvUiT_3QdMa3x-quKx6Q0WHcTXJOHOgLklO0/edit?resourcekey=0-fRNy_SLWYZUkPJfkMN3wCw&tab=t.0#bookmark=id.31h792kao2zx)